sysusers.d/systemd-coredump.conf.in

```diff
@@ -0,0 +1,10 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+{% if ENABLE_COREDUMP %}
```

Comment:
This condition seems not necessary any more.

Reply:
I've had already asked for it in https://github.com/systemd/systemd/issues/20044#issuecomment-869876506 ...

I guess one can drop it then from `systemd-remote.conf.in`, too? That's why I kept it originally, cause that had it, too.